### PR TITLE
Refactor.makara builder

### PR DESCRIPTION
**Pull request by grawk**
use makara-builder interface introduced here: https://github.com/krakenjs/makara-builder/pull/4

This pulls up the spundle and directory creation from the "writer" method and into makara-builder. Removes repeated code from this module and any other "makara-*" modules.

### CODE REVIEW

**Comment by xjamundx** [comment]
Seems like a nice change. Way simpler. Is it way faster? I remember the old one was pretty slow

**Comment by grawk** [comment]
I doubt it's any faster. I never performance tested it but any slowness is probably in the browserify bundler.

File: package.json [hunk 1]
```diff
@@ -1,25 +1,18 @@
 {
   "name": "makara-browserify",
-  "version": "1.2.3",
+  "version": "1.2.5",
```
Comment (skoranga) [inline package.json:3]: patch version change?

Comment (grawk) [inline package.json:3]: This line just brings the version up to date with the current version. When the change is merged, I will use `npm version patch` to increment the version. I will also add the final patch release version to the changelog.

**Comment by skoranga** [comment]
tests?

**Comment by grawk** [comment]
@skoranga there is already a unit test for the "build" method

**Comment by skoranga** [comment]
👍 
if make sense would love to see atleast some -ive tests?

**Comment by grawk** [comment]
I think the place for those tests would be in `makara-builder` since here the "build" method is mainly a pass through to makara-builder. I tried a few times and it's not straightforward how to actually get an error out of makara-builder (and under that, spundle). Definitely an area for further investigation. @skoranga

**Comment by subeeshcbabu-zz** [comment]
👍

**Pull request merged**